packages/metal-dom/test/dom.js

```diff
@@ -412,6 +412,92 @@ describe('dom', function() {
 			assert.strictEqual('test', listener.args[0][0].test);
 			document.body.removeChild(element);
 		});
+
+		it('should not trigger dom click event for disabled elements', function() {
+			var input = document.createElement('input');
+			var select = document.createElement('select');
+			var button = document.createElement('button');
+			var textarea = document.createElement('textarea');
+			input.disabled = true;
+			select.disabled = true;
+			button.disabled = true;
+			textarea.disabled = true;
+			document.body.appendChild(input);
+			document.body.appendChild(select);
+			document.body.appendChild(button);
+			document.body.appendChild(textarea);
+
+			var listener1 = sinon.stub();
+			var listener2 = sinon.stub();
+			var listener3 = sinon.stub();
+			var listener4 = sinon.stub();
+
+			input.addEventListener('click', listener1);
+			select.addEventListener('click', listener2);
+			button.addEventListener('click', listener3);
+			textarea.addEventListener('click', listener4);
+
+			dom.triggerEvent(input, 'click');
+			dom.triggerEvent(select, 'click');
+			dom.triggerEvent(button, 'click');
+			dom.triggerEvent(textarea, 'click');
+			assert.strictEqual(0, listener1.callCount);
+			assert.strictEqual(0, listener2.callCount);
+			assert.strictEqual(0, listener3.callCount);
+			assert.strictEqual(0, listener4.callCount);
+		});
+
+		it('should not trigger dom click event to an element which its valid parent is disabled', function() {
```

Comment:
The wording on this description sounds a bit weird in English right now, starting from the `which` word. It also makes it seem as if any parent would cause this, on any child, when it should only happen for form control elements with fieldset parents. Could you replace it with something more like:

`should not trigger dom click event on a form control with a disabled fieldset parent`

Same goes for other descriptions that are using `which` in a similar way to this.